i18n/languages.json

```diff
@@ -294,6 +297,9 @@
         "diff-title-original-content": "النص الأصلي",
         "diff-title-your-content": "نصك",
         "diff-error": "فشل تحميل الفرق",
+        "diff-patrol": "Mark as patrolled",
+        "diff-patrol-succeed": "✔",
+        "diff-patrol-fail": "Failed...Retry",

```

Comment:
You don't have to use English to fill in a language you don't speak. The missing strings will be auto fallback to English.

(The same for other languages below